lib/smartdown/parser/scenario_set_interpreter.rb

```diff
@@ -55,6 +56,10 @@ def has_exact_markers?
         @scenario_lines.any? { |line| line.match(/(exact markers:|exact marker:)/) }
       end
 
+      def has_content_to_match?
+        @scenario_lines.any? { |line| line.match(/has content:/) }
```

Comment:
I'm not sure how likely/relevant it is, but have you considered using `line.start_with?("has content:")` or `line.match(/^has content:/)`?